src/components/CldVideo/CldVideo.vue

```diff
@@ -127,7 +127,7 @@ export default {
     const poster = cldPoster ? this.posterUrl : this.getPosterUrl()
 
     return (
-      <video attrs={this.$attrs} poster={poster}>
+      <video controls attrs={this.$attrs} poster={poster}>
```

Comment:
this should probably be configurable, no?
the user might choose not to pass controls. (isn't that part of this.$attrs?)

Reply:
if user doesn't pass controls, it should be enabled by default, no? Else you will not see any video, only the poster, which make the whole idea of video component kind of redundant. I can remove this `controls` if you want to leave it for users to remember putting controls themselves

Reply:
Controls should only determine if the html element has the video controls, if we're doing something different here it sounds like an implementation bug. 

What about these situations? will these work?

No poster -> show controls?
No poster -> hide controls?
Poster -> show controls?
Poster -> hide controls?

I'm fine with it not being part of this PR, but if we need to fix this behaviour, It might be best to leave this option to the user.

Reply:
it wokrs for those situations but agreed to leave the option to user.